_translator-files/po/de/Installation-for-Windows.po

```diff
@@ -69,13 +70,13 @@ msgstr "**Downloade und installiere einen ASIO-Treiber**. Wir empfehlen, eine So
 #: ../wiki/en/Installation-for-Windows.md:18
 #, no-wrap
 msgid "[Download Jamulus]({{ site.download_root_link }}{{ site.download_file_names.windows }}){: .button}\\\\\n"
-msgstr "[Downloade Jamulus]({{ site.download_root_link }}{{ site.download_file_names.windows }}){: .button}\\\\\n"
+msgstr "[Download Jamulus]({{ site.download_root_link }}{{ site.download_file_names.windows }}){: .button}\\\\\n"
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-Windows.md:19
 #, no-wrap
 msgid "**Mirror 2:** [SourceForge](https://sourceforge.net/projects/llcon/files/latest/download)\n"
-msgstr "**Alternativ-Download:** [SourceForge](https://sourceforge.net/projects/llcon/files/latest/download)\n"
+msgstr "**Mirror 2:** [SourceForge](https://sourceforge.net/projects/llcon/files/latest/download)\n"
```

Comment:
Not sure if this is correct but it should be translated.

Reply:
Didnt translate - used original